docs/release-notes/8.0.44-35.md

```diff
@@ -0,0 +1,93 @@
+
+* [PS-9838](https://perconadev.atlassian.net/browse/PS-9838): The server’s estimate for how many rows fit in one index leaf (`guess_rec_per_key()`) uses two numbers: `used_keyparts` (the parts actually stored in the index) and `user_defined_key_parts` (the parts the user declared).
+If these two counts differ, the formula calculates the wrong capacity, leading to inaccurate index‑size estimates.
+
+* [PS-10283](https://perconadev.atlassian.net/browse/PS-10283): A debug‑assertion failure (`field.field_no < dict_index_get_n_fields(index)`) that occurred during recovery of InnoDB tables using `ROW_FORMAT=REDUNDANT`. The fix added a conditional bypass of the index‑field validation when the server was in recovery mode and the table was non‑compact (redundant), preventing the assertion from triggering and allowing the server to start and recover normally.
+
+### MyRocks
+
+* [PS-9220](https://perconadev.atlassian.net/browse/PS-9220): During slow‑query‑log rotation the server attempts to write to the base file `slow.log` (which isn’t created), raises an error - `errno 2 – No such file or directory`, but the logs are written with numeric suffixes (for example, `slow.log.000087`) that obey the configured size and file‑count limits.
+
+* [PS-9452](https://perconadev.atlassian.net/browse/PS-9452): When the server uses a thread‑pool (thread_handling = pool‑of‑threads) and the variable `thread_pool_high_prio_mode` is set to `transactions`, every new client connection is placed in the normal‑priority queue. The queue is processed only if there is a free thread in the pool.
+
+* [PS-9680](https://perconadev.atlassian.net/browse/PS-9680): Calling `ALTER TABLE … IMPORT TABLESPACE` (or any RocksDB operation that creates a new table) from many threads at the same time can trigger the error “Invalid argument: Global seqno is required, but disabled.”
+Even when `rocksdb_bulk_load` is set to `0`, this error indicates that RocksDB’s global sequence‑number tracking is turned off while the engine still expects it, causing write‑order mismatches that may corrupt the data.
+
+* [PS-9834](https://perconadev.atlassian.net/browse/PS-9834): The Audit Log Filter incorrectly deletes all rotated logs if the `audit_log_filter.max_size` is set > 2 GiB.
+
+* [PS-9840](https://perconadev.atlassian.net/browse/PS-9840): An assertion was triggered because a data dictionary table existed for table T, yet that table was not registered in RocksDB. The storage engine could not locate the corresponding physical representation, leading to the failure.
+
+* [PS-9842](https://perconadev.atlassian.net/browse/PS-9842): An assertion was raised because `rocksdb::MergingIterator::CurrentForward()` asserted that `minHeap_.empty() || minHeap_.top()->type == HeapItem::Type::ITERATOR`. The fix corrected the heap‑state handling, eliminating the SIG 6 termination and stabilized table‑check operations.
```

Comment:
The improved description:
An assertion could occur due to an invalid iterator. After this fix, Rdb_iterator_base is always invalidated when an error occurs.